lib/src/connection_checker.dart

```diff
@@ -55,14 +59,16 @@ class ConnectionChecker {
   /// connection get back. The default value is set to 4 attempts.
   factory ConnectionChecker({
     bool checkHostReachability = true,
-    List<InternetAddress>? checkAddresses,
+    List<ConnectionEntry>? checkAddresses,
```

Comment:
Line 45 and line 155 mention only about `checkAddresses`, not the `checkApiUrls`. If we want to make the docs clearer then we should either:
- also mention `checkApiUrls` next to each `checkAddresses` phrase (I only found two places where we mention `checkAddresses`)
- use only the `checkAddresses` var but describe its usage based on the current platform (web or native)